### PR TITLE
Cleanup registry unit tests

### PR DESCRIPTION
**Pull request by minghinmatthewlam**
## Why this should be merged
Cleans up `_createWarpOffChainMessage` in Teleporter registry unit tests

## How this works
Removes extra parameter `destinationAddress`, because it already takes in registry address, and should be same value on success.

## How this was tested
Unit tests, CI

## How is this documented
n/a

### CODE REVIEW

**Review by geoff-vball — APPROVED** [review]

File: contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol [hunk 1]
```diff
@@ -252,7 +230,11 @@ contract TeleporterRegistryTest is Test {
 
         // Check if we have an invalid destination address
         warpMessage = _createWarpOffChainMessage(
-            latestVersion + 1, address(this), teleporterAddress, address(teleporterRegistry)
```
Comment (cam-schultz) [inline contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol:255]: Can you help me understand why `destinationAddress` was `address(this)` previously, and why this is no longer needed?

**Review by cam-schultz — COMMENTED** [review]
One clarification question to make sure the tests are covering the same cases as before.

File: contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol (hunk 1 above)
Comment (minghinmatthewlam) [reply contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol:255]: it was set as `address(this)` to mock an invalid Warp message to the registry, we still set it in the next line

Comment (cam-schultz) [reply contracts/src/Teleporter/upgrades/tests/TeleporterRegistryTests.t.sol:255]: Thanks!

**Review by cam-schultz — APPROVED** [review]

**Review by michaelkaplan13 — APPROVED** [review]

**Pull request merged**